ComponentAssemblies/ClientComponents/StatisticsWidget/projects/StatisticsWidget.json

```diff
@@ -0,0 +1,147 @@
+  "configurationMappings" : { },
+  "configurationProperties" : { },
+  "isAssembly" : true,
+  "links" : [ ],
+  "name" : "StatisticsWidget",
+  "options" : {
+    "description" : "The Statistics Component has **title**, **value**, and **delta** fields, all programatically set. It also has an optional **Context Menu**, which can be used as a menu or to trigger other actions.",
+    "dockCollapsed" : {
+      "bottom" : false,
+      "left" : false,
+      "right" : false,
+      "top" : false,
+      "viewtabs" : false
+    },
+    "dockDimensions" : {
+      "bottom" : 200,
+      "debug" : [ 0, 0, 0 ],
+      "left" : 210,
+      "right" : 220,
+      "top" : 0,
+      "viewtabs" : 0
+    },
+    "exclusionList" : [ "client/Compound", "client/com.vantiq.clientcomponent.Compound" ],
+    "filterBitArray" : "ffffffffffffffffffffffffffffffff",
+    "isModeloProject" : true,
+    "layoutStyle" : "flex",
+    "openFolders" : [ "VIEWROOT/com.vantiq.appcomponents", "com.vantiq.appcomponents/app", "com.vantiq.clientcomponent/client", "document" ],
+    "partitionsSettings" : { },
+    "rootViewFlavor" : 5,
+    "showGrid" : true,
+    "structure" : {
+      "fraction" : 1,
+      "type" : "R[6]"
+    },
+    "type" : "dev",
+    "v" : 6,
+    "viewUUID" : "MAINVIEW"
+  },
+  "partitions" : [ ],
+  "resources" : [ {
+    "label" : "com.vantiq.appcomponents.MichaelAC",
+    "name" : "com.vantiq.appcomponents.MichaelAC",
+    "resourceReference" : "/collaborationtypes/com.vantiq.appcomponents.MichaelAC",
+    "timestamp" : "2025-02-26T23:58:41.227Z",
+    "type" : 12
+  }, {
+    "label" : "com.vantiq.clientcomponent.Statistics",
+    "name" : "com.vantiq.clientcomponent.Statistics",
+    "resourceReference" : "/system.clients/com.vantiq.clientcomponent.Statistics",
+    "timestamp" : "2025-04-07T17:07:43.478Z",
+    "type" : 72
+  }, {
+    "label" : "com.vantiq.themes.Themes",
+    "name" : "com.vantiq.themes.Themes",
+    "resourceReference" : "/system.clients/com.vantiq.themes.Themes",
+    "timestamp" : "2025-03-24T17:47:11.890Z",
+    "type" : 15
+  }, {
+    "label" : "vantiq/themes/Cornflower.thm",
+    "name" : "vantiq/themes/Cornflower.thm",
+    "resourceReference" : "/documents/vantiq/themes/Cornflower.thm",
+    "timestamp" : "2025-03-24T17:54:41.917Z",
+    "type" : 19
+  } ],
+  "selectData" : { },
+  "tools" : [ {
+    "isPinned" : false,
+    "name" : "Catalogs",
+    "pane" : {
+      "address" : "R[4]",
+      "isActive" : true,
+      "isActiveTool" : true
+    },
+    "state" : 8,
+    "type" : 115
+  }, {
+    "isPinned" : false,
+    "name" : "Documents",
+    "pane" : {
+      "address" : "R[3]"
+    },
+    "state" : 8,
+    "type" : 26
+  }, {
+    "isPinned" : false,
+    "name" : "Errors",
+    "type" : 13
+  }, {
+    "dockLocation" : "top",
+    "isPinned" : false,
+    "name" : "Inactive Panes",
+    "type" : 99
+  }, {
+    "isPinned" : false,
+    "name" : "Manage Catalog",
+    "pane" : {
+      "address" : "R[5]"
```

Comment:
remove Manage Catalog lines   (Catalogs is probably OK to leave in, but could remove that, too)